docs/reference/commandline/service_logs.md

```diff
@@ -32,6 +32,76 @@ Options:
 
 The `docker service logs` command batch-retrieves logs present at the time of execution.
 
+## Examples
+
+### Log a service
+
+```bash
+$ docker service create --name redis redis:3.0.6 
```

Comment:
Can you use `--replicas=2` to illustrate it's showing logs for all tasks that back the service?